stellar-auth/pkg/auth/auth.go

```diff
@@ -25,7 +26,7 @@ type AuthManager interface {
 	UpdatePassword(ctx context.Context, token, currentPassword, newPassword string) error
 	GetUser(ctx context.Context, tokenString string) (*User, error)
 	GetUserID(ctx context.Context, tokenString string) (string, error)
-	GetAllUsers(ctx context.Context, tokenString string) ([]User, error)
+	GetAllUsers(ctx context.Context, tokenString string, queryParams *data.QueryParams) ([]User, error)
```

Comment:
This is not the pattern we use for the `stellar-auth` package. It can't depend on the `SDP`. I suggest you create another structure in this package that receives the values you need and use it here.